log/log.go

```diff
@@ -0,0 +1,47 @@
+package log
+
+import (
+	"io/ioutil"
+
+	"github.com/Sirupsen/logrus"
+)
+
+var log *logrus.Logger
+
+const DebugLevel = logrus.DebugLevel
+
+func Debugf(format string, args ...interface{}) {
+	log.Debugf(format, args...)
```

Comment:
> I believe because of the extra layer, your wrapper will log incorrect lines, you should fix that.

~~Could you elaborate on that or give an example? Were are simply passing the arguments through.~~

I got it wrong, you mean _line numbers_, right?